api/tough-cookie.api.md

```diff
@@ -264,38 +266,38 @@ export class ParameterError extends Error {
 // @public (undocumented)
 export const parse: (str: string, options?: {
     loose?: boolean | undefined;
-} | undefined) => Cookie | null | undefined;
+} | undefined) => Cookie | undefined;
 
 // @public (undocumented)
-export function parseDate(str: string | undefined | null): Date | undefined;
+export function parseDate(str: Nullable<string>): Date | undefined;
 
 // @public (undocumented)
 export function pathMatch(reqPath: string, cookiePath: string): boolean;
 
 // @public (undocumented)
-export function permuteDomain(domain: string, allowSpecialUseDomain?: boolean): string[] | null;
+export function permuteDomain(domain: string, allowSpecialUseDomain?: boolean): string[] | undefined;
 
 // @public
 export function permutePath(path: string): string[];
 
-// @public (undocumented)
-export const PrefixSecurityEnum: Readonly<{
-    SILENT: "silent";
-    STRICT: "strict";
-    DISABLED: "unsafe-disabled";
-}>;
+// @public
+export enum PrefixSecurity {
```

Comment:
Enums are gross and it's a best practice to avoid them. What does the documentation look like if we use a readonly object?

Reply:
The field-level docs don't render in that case.  You get a page with just `export const PrefixSecurityEnum` which is kind of useless but maybe this should be explored further.  

I also think I should revert this rename since this export exists in `v4` (even though an `enum` with the suffix `Enum` offends me 😅)

Reply:
@wjhsf reverted this `enum` and naming change so the previous `PrefixSecurityEnum` as a readonly constant is retained. 

The property docs are not supported when doing this so I've moved each property description into the main description as bullet points.